src/api/subscriptions.js

```diff
@@ -236,6 +236,17 @@ export class Subscriptions {
    */
   attachButton(button, optionsOrCallback, opt_callback) {}
 
+  /**
+   * Creates smartButton element and the provided callback.
+   * The default theme is "light".
+   *
+   * @param {!Element} container
+   * @param {!ButtonOptions=} opt_options
+   * @param {function()=} opt_callback
+   * @return {!Promise}
```

Comment:
What does this "promise" mean at this time?

Reply:
Not required, removed.